Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/AbstractSqlEamDb.java

```diff
@@ -384,6 +384,46 @@ public CorrelationCase getCaseByUUID(String caseUUID) throws EamDbException {
 
         return eamCaseResult;
     }
+    /**
+     * Retrieves Case details based on Case ID
+     *
+     * @param caseID unique identifier for a case
+     *
+     * @return The retrieved case
+     */
+    @Override
+    public CorrelationCase getCaseById(int caseId) throws EamDbException {
```

Comment:
Should not be public

Reply:
Should we extend the InstanceCallback interface instead and have query results join cases by id so we can retrieve the UUID and get the case that way? Same for datasources?

Reply:
Oh yea, never mind.  You need it outside of the package and we are still keeping it from being exported from the NBM.